Bio/SearchIO/BlastIO/blast_tab.py

```diff
@@ -233,11 +246,12 @@ def _prep_fields(self, fields):
         # we want to transform 'std' to its proper column names
         if "std" in fields:
             idx = fields.index("std")
-            fields = fields[:idx] + _DEFAULT_FIELDS + fields[idx + 1:]
+            fields = fields[:idx] + _DEFAULT_FIELDS + fields[idx + 1 :]
         # if set(fields) has a null intersection with minimum required
         # fields for hit and query, raise an exception
-        if not set(fields).intersection(_MIN_QUERY_FIELDS) or \
-                not set(fields).intersection(_MIN_HIT_FIELDS):
+        if not set(fields).intersection(_MIN_QUERY_FIELDS) or not set(
+            fields
+        ).intersection(_MIN_HIT_FIELDS):
```

Comment:
An example where the black output just looks ugly - but I don't immediately see how to improve on it, so may as well leave it.